docs/cloud/integrations/javascript.md

```diff
@@ -0,0 +1,106 @@
+---
+title:  Connect to Steampipe Cloud with Javascript
+sidebar_label: Javascript
+---
+# Connect to Steampipe Cloud from Javascript
+
+The Steampipe Cloud workspace is a Postgres database, with the use of the `pg` client you can connect and query your workspace database with Javascript.
+
+The [Connect](/docs/cloud/integrations/overview) tab for your workspace provides the details you need to connect Steampipe Cloud with Javascript.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/steampipe-cloud-connect-details.jpg" />
+</div>
+
+To start, you would need to install the postgres client with `nmp i pg`, specify the connection string, establish a successful connection, run your query and get the results. In this example, we query the state and the region of `aws_vpc` using Javascript.
```

Comment:
```suggestion
You'll need the Postgres client for JavaScript, which you can install using `npm i pg` or `yarn add pg`. 
Then you specify the connection string, create a connection, run a query, and fetch results. In this example, we query the state and the region of `aws_vpc`.
```

Reply:
Update the suggested text to the document and pushed it.